timvt/events.py

```diff
@@ -11,7 +12,10 @@ def create_start_app_handler(app: FastAPI) -> Callable:  # type: ignore
     """App start event."""
 
     async def start_app() -> None:
+        global Tables
         await connect_to_db(app)
+        app.state.Catalog = Catalog(app)
+        await app.state.Catalog.init()
```

Comment:
❤️

Reply:
Figuring out that one async call took more time that I care to reveal.